static/js/whyis_vue/utilities/vega-chart.js

```diff
@@ -75,10 +76,16 @@ function buildChartLd (chart) {
 
 function extractChart (chartLd) {
   const chart = Object.entries(chartFieldUris)
-    .reduce((o, [field, uri]) => Object.assign(o, { [field]: chartLd[uri][0]['@value'] }), {})
+    .reduce((o, [field, uri]) => {
+      let value = ""
+      if (uri in chartLd) {
+        value = chartLd[uri][0]['@value']
```

Comment:
data URIs should be resources, not literals. FOAF:depiction is an object property, after all.